sydent/http/servlets/v2_servlet.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from twisted.web.resource import Resource
+
+from sydent.http.servlets import jsonwrap, send_cors
+
+
+class V2Servlet(Resource):
+    isLeaf = False
+
+    def __init__(self, syd):
+        Resource.__init__(self)
+        self.sydent = syd
+
+    @jsonwrap
+    def render_GET(self, request):
+        send_cors(request)
+        request.setResponseCode(200)
+        return {}
```

Comment:
Ermh, why do we have a no-op request?

Reply:
This file was just copied from v1. #justsydentthings

Reply:
Lets not port pointless API endpoints to v2 unless there's a good reason to :)